src/utils/evalComponentCode.js

```diff
@@ -17,20 +17,22 @@ module.exports = function evalComponentCode(code) {
     function fakeRequire() {}
     fakeRequire.default = fakeRequire
 
-
-    let requireSanbox = function (element) {
-      if (element === '@babel/runtime/helpers/interopRequireDefault') {
-        return function (value) {
+    let requireSanbox = function(element) {
+      // depending on the preset that one is running,
+      // babel will not always use the root interopRequireDefault
+      // for babel es6, it uses .../builtin/es6/interop...
+      if (/@babel\/runtime\/helpers\/.*interopRequireDefault$/.test(element)) {
```

Comment:
Looks like in latest version it's now `@babel/runtime-corejs2/helpers/interopRequireDefault` so this fix no longer works.  Changing it to `if (/@babel[\\,\/]runtime(.*)\/helpers\/.*interopRequireDefault$/.test(element)) {` would solve it.

Reply:
It would be a temporary solution. Babel’s internals are almost guaranteed to change. With AST parsing instead of actual evaluation of the documented component, no need for require anymore.

Reply:
See #68